.github/workflows/Construct.yml

```diff
@@ -0,0 +1,49 @@
+name: Build installers
+
+on:
+  workflow_call:
+
+permissions:
+  contents: write    
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  openlane-sky130a-installer:
+    runs-on: "ubuntu-20.04"
+    steps:
+      - name: checkout conda-eda
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # fetch all tags
+
+      - name: construct installer
+        run: |
+          export CI_SCRIPTS_PATH="$(pwd)/ci"
+          ./ci/install.sh
+          conda install constructor
+          constructor installer/openlane.sky130a/
+
+      - name: archive openlane.sky130a installer
+        uses: actions/upload-artifact@v3
+        with:
+          name: "openlane-sky130a-installer"
+          path: |
+            /home/runner/work/conda-eda/conda-eda/openlane-sky130a-0-Linux-x86_64.sh
+
+      - run: |
+          git config --local user.name conda-eda@bot
+          git config --local user.email conda-eda@bot
+          TAG=$(git describe --tags)
+          git tag $TAG
+          git push origin "HEAD:refs/tags/$TAG"
+          echo "TAG=$TAG" >> $GITHUB_ENV
```

Comment:
I've just realized there are two issues here:
1. `git describe --tags` will use tags from previous runs of this code in `Construct.yml` so pretty soon we'll have tags like: `v0.0-1337-gc24a15291-X-gSHA-X-gSHA-X-gSHA-X-gSHA-...`
2. There's no check if the tag already exists and the workflow will be triggered every day (with `Upload.yml` nightly runs).

Reply:
> 1. 

Oh I did had this issue, and handled it by removing `--tags` from the `git describe` command

> 2.

You're right it will overwrite the existing release with a new build even if there is no change in `conda-eda` which could be unfortunate if there are some upstream change but no `conda-eda` commit.